src/Plugins/OrientationAnalysis/src/OrientationAnalysis/Filters/GenerateFZQuaternions.cpp

```diff
@@ -267,7 +267,6 @@ Result<> GenerateFZQuaternions::executeImpl(DataStructure& dataStructure, const
   // Parallel algorithm to find duplicate nodes
   ParallelDataAlgorithm dataAlg;
   dataAlg.setRange(0ULL, static_cast<size_t>(quatArray.getNumberOfTuples()));
-  dataAlg.setParallelizationEnabled(false);
```

Comment:
Check if parallelization enabled is intended.

Reply:
That was there for a reason. Not sure it works in parallel.

Reply:
If @imikejackson  was the person who disabled the parallelization then I would assume I did it because the parallel part didn't work correctly? I can only assume that is why I did it